docker-compose.yml

```diff
@@ -1,18 +1,16 @@
 version: "3.4"
 services:
-  bind.dnp.dappnode.eth:
+  bind-slim.dnp.dappnode.eth:
     build: ./build
-    image: "bind.dnp.dappnode.eth:0.2.6"
-    container_name: DAppNodeCore-bind.dnp.dappnode.eth
+    image: "bind-slim.dnp.dappnode.eth:0.2.6"
+    container_name: DAppNodeCore-bind-slim.dnp.dappnode.eth
     restart: always
     networks:
-      network:
+      dncore_network:
         ipv4_address: 172.33.1.2
     logging:
       driver: journald
+
 networks:
-  network:
-    driver: bridge
-    ipam:
-      config:
-        - subnet: 172.33.0.0/16
+  dncore_network:
+    external: true
```

Comment:
We were debating some time ago about how to handle the dncore_network. By declaring it as external you are expecting that this package will be started strictly after another that creates the network (i.e. the DAPPMANAGER). What do you think? Should we change the dncore_network setup in another PR?

Reply:
This is the main reason PR is still a draft. That change is just for development purposes so when I compose up it joins proper network.